arboretum/common/kube_utils.py

```diff
@@ -0,0 +1,44 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common Kubernetes functions."""
+
+from requests import HTTPError
+
+
+def get_cluster_resources(session, token, resource_types, verify):
+    """Get resource from cluster.
+
+    :param requests.Session session: a requests.session object
+    :param str token: Bearer token
+    :param list(str) resource_types: list of resource type in plural form
+                                     (e.g., "pods") or custom resource API
+                                     name (e.g.,
+                                     "apigroup.example.com/v1/mycustom")
+    :param str verify: Path for CA certificate file, or False if skip TLS
+                       server certificate verification (same as "curl -k")
```

Comment:
This can be either a string or a boolean so labeling it a string is misleading.

```suggestion
    :param verify: path to a CA certificate, or True/False. Set to False to 
      skip TLS server certificate verification.  Defaults to True.
```